crates/bevy_core_pipeline/src/oit/oit_draw.wgsl

```diff
@@ -0,0 +1,34 @@
+#define_import_path bevy_core_pipeline::oit
+
+#import bevy_pbr::mesh_view_bindings::{view, oit_layers, oit_layer_ids, oit_layers_count}
+
+#ifdef OIT_ENABLED
+// Add the fragment to the oit buffer
+fn oit_draw(position: vec4f, color: vec4f) -> vec4f {
```

Comment:
Rather than discard, you could also just not output any fragments? You can have a fragment shader that doesn't output anything if you're just using atomic writes, that's what meshlets do.

Reply:
Yeah, I don't remember why I used discard. I know I experimented with both but not sure why I used discard here. I'll revert it back to returning 0 color.